modules/network-observability-ebpf-rule-flow-filter.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-ebpf-flow-rule-filter_{context}"]
+= eBPF flow rule filter
+You can use rule-based filtering to control the volume of packets cached in the eBPF flow table. For example, a filter can specify that only packets coming from port 100 should be recorded. Then all packets which do not match the filter are not cached and only the ones that match are cached.
+
+== How ingress and egress traffic filtering works
```

Comment:
Your module titles need an anchor id and the `_{context}` variable. This needs to be addressed in a few places, but this is the only place I will call it out. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-in-module-files

Reply:
Ah dangit I forgot that even 2nd level headers need anchor id.